### PR TITLE
[PCover] Add CI

### PR DESCRIPTION
**Pull request by aman-goel**
Updates include:

[PCover]
- Add GitHub action for explicit-state checker
- Improve error reported due to detected cycles in a schedule
- Cleanup when liveness checks are enforced

[PSym]
- Improve fix point detection (enabled via `--state-caching exact` CLI option)
- Minor correction to corner-case symbolic set comparison

### CODE REVIEW

File: Src/PRuntimes/PSymRuntime/src/test/java/psym/TestSymbolicRegression.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public class TestSymbolicRegression {
   private static final String outputDirectory = "output/testCases";
   private static final List<String> excluded = new ArrayList<>();
   private static String mode = "verification";
-  private static String runArgs = "--seed 0 --timeout 60 --max-steps 300";
+  private static String runArgs = "--seed 0 --timeout 60 --iterations 100 --max-steps 300";
```
Comment (ankushdesai) [inline Src/PRuntimes/PSymRuntime/src/test/java/psym/TestSymbolicRegression.java:29]: why is the seed fixed?

**Review by ankushdesai — APPROVED** [review]
